content_schemas/formats/homepage.jsonnet

```diff
@@ -6,5 +6,7 @@
       properties: {},
     },
   },
-  links: {},
+  links: {
+    link_collection: "Collection of links to be used to display popular links on homepage"
```

Comment:
I would prefer this link to still be called `popular_links` I think - it doesn't have to match the document type.

The difference is between:

```
{
  "base_path": "/",
  "document_type": "homepage",
  "links": {
    "link_collection": [
      {
        "document_type": "link_collection",
        ...
```

and:


```
{
  "base_path": "/",
  "document_type": "homepage",
  "links": {
    "popular_links": [
      {
        "document_type": "link_collection",
        ...
```

Reply:
Ah ok, thats interesting, make sense.